pkg/provider/pulumi/aws/aws.go

```diff
@@ -475,6 +477,14 @@ func (a *awsProvider) Deploy(ctx *pulumi.Context) error {
 			return err
 		}
 
+		_, ok := uniquePolicies[policyName]
+		if ok {
+			// no point creating multiple policies with the same content.
```

Comment:
We should probably find out what caused this, it could be a mistake in the project that we could highlight as well as adding this protection.

Reply:
I think logging an error would be good if we can highlight this as part of our output logging during deployment.

Reply:
mmm, let's get this merged and i'll try and do that..